ci/pipeline.yml

```diff
@@ -346,12 +345,10 @@ resources:
     deployment: domain-broker
 
 - name: common-production
-  type: cg-common
+  type: s3-iam
   source:
-    bucket_name: ((secrets-bucket-production))
-    secrets_file: domain-broker-production.yml
-    secrets_passphrase: ((secrets-passphrase-production))
-    bosh_cert: bosh.pem
+    bucket: ((secrets-bucket-production))
+    versioned_file: domain-broker-production-decrypted.yml
     region: ((aws-region))
```

Comment:
Looks like this should be `region_name`: https://github.com/alphagov/paas-s3-resource#source-configuration